packages/core/src/components/button/abstractButton.tsx

```diff
@@ -75,22 +75,24 @@ export abstract class AbstractButton<T> extends React.Component<React.HTMLProps<
         };
     }
 
-    protected handleKeyDown = (e: React.KeyboardEvent<HTMLElement>) => {
+    protected handleKeyDown = (e: React.KeyboardEvent<any>) => {
```

Comment:
// CC @giladgray, who looked at this with me before committing

I kept seeing compiler errors to the effect of `KeyboardEvent<T> is not a supertype of KeyboardEvent<HTMLElement>` for the `safeInvoke` line below. If there's a better way to address that issue, let me know. 👍